pkgs/by-name/st/structurizr/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchurl, unzip, jre }:
+
+stdenv.mkDerivation rec {
+  pname = "structurizr-cli";
+  version = "v2025.05.28";
+
+  src = fetchurl {
+    url = "https://github.com/structurizr/cli/releases/download/v${version}/structurizr-cli.zip";
+    sha256 = "1iddrn56vadf7mryc2gicg99aknw84bgwmn01dg7awn2vixpzs5v";
+  };
+
+  nativeBuildInputs = [ unzip ];
+  buildInputs = [ jre ];
+
+  installPhase = ''
+    # unzip
+    mkdir -p $out/bin
+    unzip $src -d tmp
+    cp -r tmp/* $out/
+    # own wrapper script since we use our own java
+    cat > $out/bin/structurizr-cli <<EOF
+    #!${stdenv.shell}
+    for jar in $out/lib/*.jar; do
+      CLASSPATH="\$CLASSPATH:\$jar"
+    done
+    exec ${jre}/bin/java -cp "\$CLASSPATH" com.structurizr.cli.StructurizrCliApplication "\$@"
+    EOF
+
+    chmod +x $out/bin/structurizr-cli
+  '';
+
+  meta = with lib; {
```

Comment:
Adding some sort of `installCheckPhase` would be good since there is no checkPhase. Whether that is a `versionCheckHook` or what.